processing/src/main/java/org/apache/druid/query/aggregation/ExpressionLambdaBufferAggregator.java

```diff
@@ -21,73 +21,95 @@
 
 import org.apache.druid.math.expr.Expr;
 import org.apache.druid.math.expr.ExprEval;
+import org.apache.druid.math.expr.ExprType;
 
 import javax.annotation.Nullable;
 import java.nio.ByteBuffer;
 
 public class ExpressionLambdaBufferAggregator implements BufferAggregator
 {
+  private static final short NOT_AGGREGATED_BIT = 1 << 7;
+  private static final short IS_AGGREGATED_MASK = 0x3F;
+  private static final byte TYPE_MASK = 0x0F;
```

Comment:
is it possible to drop either `TYPE_MASK` or `IS_AGGREGATED_MASK` and use a common mask whose value is `0x0F` ?